examples/getting_started/react/README.md

```diff
@@ -0,0 +1,20 @@
+# Getting started
+
+```
+yarn install
+```
+
+## Setting up a server
+
+To get started, you will need to set up a GraphQL server. 
+
+You can use the [Graphback runtime example](https://github.com/aerogear/graphback/tree/master/examples/runtime-example), to get
```

Comment:
Question: when I go to that example I don't see the tasks model, I see a user model down there. Are there additional steps needed to get it working?

Reply:
For flexibility we might simply have model schema inside here and maybe use testx instead ?